apps/dialtone-documentation/docs/.vuepress/exampleComponents/ExampleTabs.vue

```diff
@@ -1,54 +1,69 @@
 <template>
-  <div class="d-d-flex d-fd-column d-w100p d-stack16">
+  <dt-tab-group
+    :size="size"
+    :inverted="inverted"
+    :borderless="borderless"
+    :disabled="disabled"
+  >
+    <template #tabs>
+      <dt-tab
+        id="1"
+        panel-id="2"
+        selected
+      >
+        <p>
+          First tab
+        </p>
```

Comment:
Set the inner content to inverted as well.

![Screenshot 2024-11-25 at 3 16 54 PM](https://github.com/user-attachments/assets/1a77eb46-c96b-4a21-805a-cb6662530191)